GOVERNANCE.md

```diff
@@ -0,0 +1,108 @@
+# Process for becoming a maintainer

```

Comment:
```suggestion
## Process for becoming a maintainer
```

Reply:
This does not look like a first level header to me. Maybe we can have "TiKV Project Governance" as the first level header, and add a general description like "The TiKV project is governed by its collaborators, specifically the senior maintainers, who are responsible for high-level guidance of the project."

Reply:
I agree, great idea. :)

Reply:
@Hoverbear I see you resolved this conversation but it seems you haven't made the change we mentioned above, which you also agreed. :)